packages/calcite-components/src/components/autocomplete-item/resources.ts

```diff
@@ -6,6 +6,7 @@ export const CSS = {
   contentCenter: "content-center",
   description: "description",
   heading: "heading",
+  textMatch: "text-match",
```

Comment:
Should this class go into the `text` util too?

Reply:
done!